change/@ni-nimble-components-d62c8a19-26e8-448d-831b-ddc56969c9bb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
In order to submit you'll still need to:
1. fix the failing unit test
2. review the Chromatic visual diffs for the wafermap stories and, if they look good, mark them as Approved.

You can see more info about these in the Status Checks section on the Conversations tab of this PR. 

![image](https://user-images.githubusercontent.com/10500124/207383729-d578fe48-e9fa-4536-b059-3ce23cab226c.png)

Reply:
Every pipeline passed I also review-ed the wafermap stories for Chromatic.